frontend/apps/admin/src/client/client.ts

```diff
@@ -50,7 +54,7 @@ export default class Client implements ApiClient {
       headers = options?.headers;
     }
 
-    return axios
+    return this.axiosInstance
```

Comment:
Here, we were ignoring the passed `axiosInstance` entirely. So injecting the token as an interceptor had no effect